test/printf-test.cc

```diff
@@ -518,3 +517,43 @@ void check_format_string_regression(fmt::string_view s, const Args&... args) {
 TEST(PrintfTest, CheckFormatStringRegression) {
   check_format_string_regression("%c%s", 'x', "");
 }
+
+TEST( PrintfTest, VSPrintfMakeArgsExample ) {
```

Comment:
nit: spaces after `(` and before `)` are not needed

Reply:
I removed all the superfluous spaces clang formatting introduced. I think I got them all.